python/fastsim/demos/__init__.py

```diff
@@ -0,0 +1,4 @@
+print(
+    "Module `fastsim.demos` has been imported.  This should only " 
+    + "happen during testing so you might be doing something wrong."
+)
```

Comment:
```suggestion
import logging

logger = logging.getLogger(__name__)
logger.warning(
    "Module `fastsim.demos` has been imported. This should only "
    + "happen during testing, so you might be doing something wrong."
)
```

Reply:
Results in something like this:
```
2023-11-02 14:23:35.117 | __init__.py:4 | WARNING: Module `fastsim.demos` has been imported. This should only happen during testing, so you might be doing something wrong.
```

Reply:
This allow lets this message be silenced by the logging level utilities in `utilities.py`